admin/sql/messaging/CreateTriggers.sql

```diff
@@ -0,0 +1,11 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+CREATE TRIGGER b_ins_hidden_message BEFORE INSERT ON hidden_message
+    FOR EACH ROW EXECUTE PROCEDURE ensure_editor_is_connected_to_message();
```

Comment:
One trigger should suffice (using `INSERT OR UPDATE`). While I don't see any triggers named e.g. `b_ins_or_upd_*`, the ones that do apply to multiple operations seem to all use the function name, so I guess we can do that.
```suggestion
CREATE TRIGGER ensure_editor_is_connected_to_message BEFORE INSERT OR UPDATE ON hidden_message
    FOR EACH ROW EXECUTE PROCEDURE ensure_editor_is_connected_to_message();
```